nixos/modules/system/boot/stage-1.nix

```diff
@@ -560,6 +560,23 @@ in
       description = "Names of supported filesystem types in the initial ramdisk.";
     };
 
+    boot.initrd.verbose = mkOption {
+      default = true;
+      type = types.bool;
+      description =
+        ''
+          Verbosity of the initrd. Please note that disabling verbosity only
+          removes the mandatory messages generated by our scripts. For a fullly
+          silent boot you might want to set the two following configuration
+          options additionally:
```

Comment:
```suggestion
          Verbosity of the initrd. Please note that disabling verbosity removes 
          only the mandatory messages generated by the NixOS scripts. For a 
          completely silent boot, you might also want to set the following 
          configuration options:
```

Reply:
Yes, it is just cosmetical :) but it is way better IMHO.

Reply:
commited, thanks!